Find a functionally equivalent code:android.widget.TextView.getExtendedPaddingTop:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the extended top padding of the view, including both the
 * top Drawable if any and any extra space to keep more than maxLines
 * of text from showing.  It is only valid to call this after measuring.
 */

@@ -2,7 +2,10 @@
     if (mMaxMode != LINES) {
         return getCompoundPaddingTop();
     }
+    if (mLayout == null) {
+        assumeLayout();
+    }
     if (mLayout.getLineCount() <= mMaximum) {
         return getCompoundPaddingTop();
     }

